src/node.cc

```diff
@@ -800,6 +800,70 @@ int ProcessGlobalArgs(std::vector<std::string>* args,
 
 static std::atomic_bool init_called{false};
 
+static ExitCode ProcessEnvFiles(std::vector<std::string>* argv,
```

Comment:
This is not the place to move this file to and this makes things overly complicated. IMHO I'm against this change.

Reply:
I'm not a huge fan of *how* it works... with the ordering issues. If you have any ideas in that department, I'd love to hear them.

I'll move this file too